src/sys/unix.rs

```diff
@@ -1449,13 +1449,13 @@ fn in_addr_convertion() {
     let raw = to_in_addr(&ip);
     // NOTE: `in_addr` is packed on NetBSD and it's unsafe to borrow.
     let a = raw.s_addr;
-    assert_eq!(a, 127 | 1 << 24);
+    assert_eq!(a.to_ne_bytes(), [127, 0, 0, 1]);
```

Comment:
The test is about checking the value of `a`, so could you move the conversion to the expected value?

Reply:
So, changing this line to something like `assert_eq!(a, u32::from_ne_bytes([127, 0, 0, 1]))?

Reply:
Yes that should work.

Reply:
Done, I've changed both the asserts to compare against plain `a`.